handbook.md

```diff
@@ -0,0 +1,233 @@
+*    Therefore for more than one parameters use `_("Params: %{a} and %{b}") % {:a => foo, :b => bar}`
+*    Note that parameters are not translated. Variables should not contain text that needs translation.
+*    To mark something for translation (but not translate) use `N_("String")`
+*    To use plural form use `n_("One", "Two", number)` - note this function always accepts three parameters as the base language is usually English but translators are able to define as many plural forms as they need.
+*    Plural forms are usually used with one parameter, do not forget to add trailing parameter to it: `n_("%s minute", "%s minutes", @param) % @param`
+*    When using strings with ERB (e.g. for deface gem), escape properly: `n_('Test <%%= %{var1} %%>') % { :var1 => "xxx" }`
+*    Note that gettext does not extract from interpolation - this will not work: `"blah #{_('key')} blah"`
+
+#### JavaScript
+
+* Both `__` and `n__` functions are available and work in much the same way as in Ruby, with the following exceptions..
+* Interpolation of values uses `Jed.sprintf` on the translated string, so for a single parameter: `Jed.sprintf(__("Foo: %s"), "bar")`
+* Multiple parameters must be named: `Jed.sprintf(__("Example: %(foo) %(bar)"), {foo: "a", bar: "b"}) `
+
+## Code reviews
+
+We review pull requests on GitHub. Most projects have one or two maintainers that review pull requests. Everyone is invited to review code.
+
+### How to become a reviewer
+
+Start reviewing patches. Simple as that.
+
+### Contributors
+
+Write a good description, a good title, and explain why change is necessary. Just a line or two explaining the problem, the solution, and a way to reproduce the issue (extra points if it's a script) help enormously.
+
+Assume reviewers have no idea or background about your patch. Even if the usual reviewers know you personally and you know they know why your change is necessary, maybe not all reviewers are aware of it. Furthermore, after some time, they might not remember well, and new reviewers will not be able to review your code without background and a good explanation.
+
+Generally, if you want to submit significant changes to the code, discuss it first on #theforeman-dev (Freenode) or Foreman-dev (mailing-list). If you know who are the usual maintainers for the code you want to change, try to ask them to validate your assumptions, your design, and if they can, ask them to review your code. This will save you a lot of going back and forth with reviewers that do not understand the reasoning behind your pull request.
+
+Submit changes incrementally. If you are submitting a pull request that will break compatibility with older APIs, spend the effort to make it compatible first. If you think your feature is not ready for prime time yet, it's OK to submit small changes and hide the feature using feature flags.
+
+Feel free to bring the attention of reviewers by calling them using '@' on GitHub.
+
```

Comment:
Might be worth mentioning the old saying "you are not your code". Reviewers *will* be consistent with style and good practice, and it's important to feel like you're being criticised personally when that happens.